Classes/Emogrifier.php

```diff
@@ -323,6 +325,16 @@ public function disableInvisibleNodeRemoval()
     }
 
     /**
+     * Disables the removal of elements with `display: none` properties.
+     *
+     * @return void
+     */
+    public function disableCharsetFix()
```

Comment:
We need unit tests for this method.

(Or, I'd prefer to always have this new behavior, and then we don't need that switch.)